inc/inventory/request.class.php

```diff
@@ -444,13 +434,34 @@ public function inError() {
     * @return array
     */
    public function getInventoryStatus(): array {
-      $item = $this->inventory->getItem();
-      return [
-         'deviceid' => $this->deviceid,
-         'itemtype' => $item->getTypeName(),
-         'items_id' => $item->getID(),
-         'metadata' => $this->inventory->getMetadata()
+      $items = $this->inventory->getItems();
+      $status = [
+         'metadata' => $this->inventory->getMetadata(),
+         'items'    => $items
       ];
+
+      if (count($items) == 1) {
+         $item = $items[0];
+         $status += [
+            'itemtype' => $item->getType(),
+            'items_id' => $item->fields['id']
+         ];
+      } else if (count($items)) {
+         $itemtype = null;
```

Comment:
A comment here can be usefull
```suggestion
         // Defines 'itemtype' only if all items has same type
         $itemtype = null;
```